int8    servo_id
float32 temp
float32 voltage
uint8   alarm
